treeherder/webapp/api/bugzilla.py

```diff
@@ -20,6 +20,12 @@ def create_bug(self, request):
                             status=HTTP_400_BAD_REQUEST)
 
         params = request.data
+
+        if params.get("crash_signature") is not None:
+            if len(params.get("crash_signature")) > 2048:
```

Comment:
These two conditions can be combined, no need for nested if statements.
Also, unless you specifically need to catch the empty string, `if foo` is easier to read than `if foo is not None`.

I'd use something like:
```python
crash_signature = params.get("crash_signature")
if crash_signature and len(crash_signature) > 2048:
```


(We can't just use `if "crash_signature" in params and len(params["crash_signature"]) > 2048:` since `crash_signature` could be None as well as missing).

I'd also add a comment saying why we're limiting to 2048.